src/Document.php

```diff
@@ -71,57 +71,73 @@ public function toDomDocument()
     }
     
     /**
-     * Create "Ad" section on "VAST" node
+     * Create inline Ad section
      *
-     * @param string $type
+     * @return \Sokil\Vast\Ad\InLine
+     */
+    public function createInLineAdSection(\DOMElement $domEl = null)
+    {
+        return $this->createAdSection($this->createAdSectionDomEl('InLine'));
+    }
+    
+    /**
+     * Create Wrapper Ad section
      *
-     * @throws \InvalidArgumentException
+     * @return \Sokil\Vast\Ad\Wrapper
+     */
+    public function createWrapperAdSection()
+    {
+        return $this->createAdSection(
+            $this->createAdSectionDomEl('Wrapper')
```

Comment:
createAdSectionDomEl<ement>. Code must be readable ) IDE will help to autocomplete long names